examples/subprocess_shell.py

```diff
@@ -5,6 +5,9 @@
 def Popen(*args, **kwargs):
     print('hi')
 
+    def __len__(self):
```

Comment:
Is this related to this patch?  Is this needed?

Reply:
Any object with length 0 is False, I set this in the test because this will not be detected as False, so is a known false positive.